docs/coverage-reporter/index.md

```diff
@@ -289,7 +289,8 @@ Follow these instructions to validate that your coverage setup is working correc
             Coverage was uploaded for a commit that belongs to a branch that isn't analyzed by Codacy.
         </td>
         <td>
-            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch or target branch for pull requests is enabled on Codacy</a>.</p>
+            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>.</p>
+            <p>Alternatively, make sure that a pull request for the branch is open and that the target branch is enabled on Codacy.</p>
```

Comment:
A couple of suggestions:

* I propose that we include the two alternative solutions for making sure that Codacy is analyzing the branch in the same paragraph
* The instruction can be more imperative, telling the user to "open a pull request" themselves, instead of making sure that a pull request is open

What do you think?

```suggestion
            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>. Alternatively, open a pull request for the branch so that Codacy starts analyzing the branch automatically (as long as the target branch is enabled on Codacy).</p>
```